go/teams/handler.go

```diff
@@ -284,9 +284,19 @@ func HandleOpenTeamAccessRequest(ctx context.Context, g *libkb.GlobalContext, ms
 			default:
 				return fmt.Errorf("Unexpected role to add to open team: %v", joinAsRole)
 			}
+
+			existingUV, err := team.UserVersionByUID(ctx, uv.Uid)
+			if err == nil {
+				if existingUV.EldestSeqno > uv.EldestSeqno {
+					return fmt.Errorf("newer version of user %v already exists in team %q (%v > %v)", tar, team.Name(), existingUV.EldestSeqno, uv.EldestSeqno)
```

Comment:
Maybe we should `continue` here instead, avoiding the case when one bad user blocks everyone else

Reply:
There are two other errors above it that are returned.

And we have a ticket to keep going on errors, so I was leaving it in here so that that ticket can handle collecting the errors and deciding what to do with them.

Reply:
OK, you're right.